Exercise_1.java

```diff
@@ -1,35 +1,65 @@
-class Stack { 
-    //Please read sample.java file before starting.
-  //Kindly include Time and Space complexity at top of each file
+// Time Complexity : O(1)

```

Comment:
Kindly mention the complexity for each function separately like for push() -O(1) , pop() - O(1)